test-crawler/workflow_scan.py

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+import requests
+import json
+from global_settings import GITHUB_API_PARAMETER
+from failure_log_crawler import FailureLogCrawler
+
+
+class WorkFlowScaner:
+    headres = {}
+    runs_len = 0
+    in_progress_num = 0
+    success_num = 0
+    failure_num = 0
+    failure_id = []
```

Comment:
Unless these are intended to be used as class-variables (which I guess they are not), I would suggest not defining them at class scope.

I understand that they are acting as default values here but I would avoid this pattern in general as it can lead to subtle bugs. See https://stackoverflow.com/questions/2681243/how-should-i-declare-default-values-for-instance-variables-in-python